chains/arbitrum-goerli-testnet.json

```diff
@@ -2,7 +2,11 @@
   "name": "Arbitrum testnet",
   "alias": "arbitrum-goerli-testnet",
   "id": "421613",
-  "symbol": "testETH",
+  "nativeCurrency": {
+    "name": "Arbitrum Goerli Ether",
```

Comment:
I find this field a bit silly, can we just call it `X` when the symbol is `X` and `Testnet X` when the symbol is `testX`?

Reply:
We could actually drop this field entirely and go back to `symbol` at the top level if we want to follow that pattern. When generating the viem compatible chain (`defineChain`) and for chains where `testnet: true`, it strips off the `test` prefix from the symbol and adds `"Testnet ". i.e. `testETH` becomes `Testnet ETH` automatically. I'll admit I'm not the biggest fan of the `nativeCurrency` object.

The only thing we need which doesn't currently exist is `decimals`

I'm also thinking of adding validation to ensure that for all chains where `testnet: true`, the symbol is prefixed with `test`. What do you think?

Reply:
I guess we just need to decide if the pattern of having symbols prefixed with `test` is a pattern we want to enforce for all chains

Reply:
> We could actually drop this field entirely and go back to symbol at the top level if we want to follow that pattern

Sounds good

> The only thing we need which doesn't currently exist is decimals

I doubt that we will ever have a chain with non-18 decimals

> I'm also thinking of adding validation to ensure that for all chains where testnet: true, the symbol is prefixed with test. What do you think?

Sounds good

> I guess we just need to decide if the pattern of having symbols prefixed with test is a pattern we want to enforce for all chains

I think that's a safe assumption

Reply:
> I doubt that we will ever have a chain with non-18 decimals

Good point. For some reason, I thought there were a couple with < 18 decimals. I considered removing `decimals` and hardcoding it to 18 now, but this might lead to unintended bugs later on. I think it's safer to leave it in

Reply:
I was suggesting that we hardcode it to 18 but they're equivalent solutions anyway